### ISSUE TITLE
Are duplicated purpose tools interesting?

### ISSUE THREAD

**Issue by eugenekolo** [zardus/ctf-tools#139]
It seems this already has Ropper, rp++, and other tools to create rop gadgets.

So, would it be interesting to add ropgadget? (my go to tool) Or, do you want to minimize duplication of tools for the job?

**Comment by zardus** [comment]
Yes! My goal is to not make any judgements about the relative usefulness of tools. It'd definitely be awesome to add ropgadget.

**Comment by zardus** [comment]
Closing, as ropgadget is now in the debian repos (and let's not duplicate effort with the debian team).
